applications/Unity.GrantManager/src/Unity.GrantManager.Application/GrantApplications/GrantApplicationAppService.cs

```diff
@@ -161,25 +168,78 @@ public async Task AddAssignee(Guid[] applicationIds, string AssigneeKeycloakId,
                 try
                 {
                     var application = await _applicationRepository.GetAsync(applicationId);
-                    if (application != null)
+                    var assignees = getAssignees(applicationId);
```

Comment:
Think this was like this before but rather PascalCase for method names, and ideally private GetAssigneesAsync which should use async await and not .Result

Reply:
Renamed to GetAssigneesAsync and refactored surounding code